layers/core_checks/cc_ray_tracing.cpp

```diff
@@ -2097,3 +2097,389 @@ bool CoreChecks::ValidateDeferredOperation(VkDevice device, VkDeferredOperationK
+        } else {
+            if (!invalid_input && command_infos.input.opType != VK_CLUSTER_ACCELERATION_STRUCTURE_OP_TYPE_MOVE_OBJECTS_NV) {
+                BufferAddressValidation<1> dst_implicit_size_validator = {{{
+                    {"VUID-VkClusterAccelerationStructureCommandsInfoNV-opMode-10467",
+                     [&accelerationStructure_size, this](const vvl::Buffer &buffer_state) {
+                         VkMemoryRequirements mem_requirements;
+                         DispatchGetBufferMemoryRequirements(device, buffer_state.VkHandle(), &mem_requirements);
+                         return mem_requirements.size < accelerationStructure_size.accelerationStructureSize;
+                     },
+                     [&accelerationStructure_size]() {
+                         return "If input::opMode is VK_CLUSTER_ACCELERATION_STRUCTURE_OP_MODE_IMPLICIT_DESTINATIONS_NV and "
+                                "input::opType is not VK_CLUSTER_ACCELERATION_STRUCTURE_OP_TYPE_MOVE_OBJECTS_NV"
+                                ", the memory in dstImplicitData must be equal to or larger than the "
+                                "VkAccelerationStructureBuildSizesInfoKHR::accelerationStructureSize value(" + std::to_string(accelerationStructure_size.accelerationStructureSize) + ")returned from "
+                                "vkGetClusterAccelerationStructureBuildSizesNV with same input parameters:";
+                     },
+                     [&accelerationStructure_size, this](const vvl::Buffer &buffer_state) { 
+                         VkMemoryRequirements mem_requirements;
+                         DispatchGetBufferMemoryRequirements(device, buffer_state.VkHandle(), &mem_requirements);
+                         return "buffer size " + std::to_string(mem_requirements.size) + 
+                                " is less than required acceleration structure size " + std::to_string(accelerationStructure_size.accelerationStructureSize); 
+                     }},
+                }}};
+
+                skip |= dst_implicit_size_validator.ValidateDeviceAddress(*this, command_infos_loc.dot(Field::dstImplicitData),
+                                                                         LogObjectList(device), command_infos.dstImplicitData);
+            }
+        }
+    }
+
+    if (command_infos.input.opMode == VK_CLUSTER_ACCELERATION_STRUCTURE_OP_MODE_COMPUTE_SIZES_NV) {
+        if (command_infos.dstSizesArray.deviceAddress == 0) {
+            skip |= LogError("VUID-VkClusterAccelerationStructureCommandsInfoNV-opMode-10470", device,
+                             command_infos_loc.dot(Field::dstSizesArray).dot(Field::deviceAddress),
+                             "(0x%" PRIx64 ") must be a valid address If input::opMode is VK_CLUSTER_ACCELERATION_STRUCTURE_OP_MODE_COMPUTE_SIZES_NV", command_infos.dstSizesArray.deviceAddress);
```

Comment:
```suggestion
                             "(0x%" PRIx64 ") must be a valid address if input::opMode is VK_CLUSTER_ACCELERATION_STRUCTURE_OP_MODE_COMPUTE_SIZES_NV", command_infos.dstSizesArray.deviceAddress);
```

same with more below